tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_orientation_zoom.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.tests.graphics.orientation_zoom_base import OrientationZoomBase
+
+
+class TestGfxSmokeTestOZ(OrientationZoomBase):
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+        self.data_layer.connect_to_wifi()
```

Comment:
oh certainly, i just changed this line to use local network (and all other similar lines) per your previous comment.